idea/Hublsoft.Task.Web/Views/LoginMonitor/Index.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model IEnumerable<Hublsoft.Web.Models.LoginMonitorModel>
+
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>Index</h1>
+
+<p>
+    <a asp-action="Create">Create New</a>
```

Comment:
This doesn't do anything does it?

Reply:
Looks like this page is just some sort of template with data passed in - not actually functional